src/main/js/posts/post.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import Comment from "./comments/comment";
-import Like from "./comments/like";
+import Like from "./likes/like";
```

Comment:
Moved the `Like` class into it's own folder to maintain consistent folder structure